interview-questions.md

```diff
@@ -9,59 +9,67 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: The `this` command is used to call methods that are in the same string. Instead of calling to the class and method, it allows coders to shortcut the code.
 
-  Researched answer:
+  Researched answer: `this` refers to an object and depending on what type of objects it is, changes how `this` keyword is called.
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is a application created by Facebook to handle a large amount of users and their user interface. The reason you would use it is it's ability to quickly refresh a page when new inputs are added by other users.
```

Comment:
The page is not necessarily refreshing, only the parts on the page that has been modified are updated when working with React as a result of the virtual DOM.